base/math.jl

```diff
@@ -536,38 +536,56 @@ Stacktrace:
 [...]
 ```
 """
-hypot(x::Number, y::Number) = hypot(promote(x, y)...)
-function hypot(x::T, y::T) where T<:Number
-    ax = abs(x)
-    ay = abs(y)
-    if ax < ay
-        ax, ay = ay, ax
-    end
+function hypot(x::Number, y::Number)
+	#= Keep specialized two argument version, because it is faster
+	than generic varargs version below =#
+	ay, ax = minmax(abs(x), abs(y))
```

Comment:
fix indenting here?

Reply:
Also below… in general we indent by 4 spaces.

Reply:
Done. Thanks.